pandas/tests/frame/test_analytics.py

```diff
@@ -507,19 +501,20 @@ def test_nunique(self):
         tm.assert_series_equal(df.nunique(axis=1, dropna=False),
                                Series({0: 1, 1: 3, 2: 2}))
 
-    def test_sum(self):
-        self._check_stat_op('sum', np.sum, has_numeric_only=True,
+    def test_sum(self, float_frame_with_na, mixed_float_frame,
+                 float_frame, float_string_frame):
+        self._check_stat_op('sum', np.sum, float_frame_with_na, float_frame,
+                            float_string_frame, has_numeric_only=True,
                             skipna_alternative=np.nansum)
 
         # mixed types (with upcasting happening)
         self._check_stat_op('sum', np.sum,
-                            frame=self.mixed_float.astype('float32'),
-                            has_numeric_only=True, check_dtype=False,
-                            check_less_precise=True)
+                            mixed_float_frame.astype('float32'), float_frame,
+                            float_string_frame, has_numeric_only=True,
+                            check_dtype=False, check_less_precise=True)
 
-    @pytest.mark.parametrize(
-        "method", ['sum', 'mean', 'prod', 'var',
-                   'std', 'skew', 'min', 'max'])
+    @pytest.mark.parametrize('method', ['sum', 'mean', 'prod', 'var',
```

Comment:
in follow, can use these fixtures (may need to make small changes) when #22762 is merged